pageserver/src/tenant/mgr.rs

```diff
@@ -80,6 +82,7 @@ pub async fn init_tenant_mgr(
     let ctx = RequestContext::todo_child(TaskKind::Startup, DownloadBehavior::Warn);
 
     loop {
+        // TODO resume deletion.
```

Comment:
Is this still valid? Could expand on what is missing?

Reply:
[e2a0938](https://github.com/neondatabase/neon/pull/4855/commits/e2a0938ae664271d4d86e2dcc04458b824a1bcc8) resolving

Reply:
Thanks! Pushed commit that removes it but forgot to reply here :)